backend/core/src/applications/applications.controller.ts

```diff
@@ -30,13 +30,25 @@ import {
   PaginatedApplicationDto,
 } from "./dto/application.dto"
 import { Expose, Transform } from "class-transformer"
-import { IsBoolean, IsOptional, IsString } from "class-validator"
+import { IsBoolean, IsOptional, IsString, IsIn } from "class-validator"
 import { PaginationQueryParams } from "../shared/dto/pagination.dto"
 import { ValidationsGroupsEnum } from "../shared/validations-groups.enum"
 import { defaultValidationPipeOptions } from "../shared/default-validation-pipe-options"
 import { applicationPreferenceExtraModels } from "./entities/application-preferences.entity"
 import { ApplicationCsvExporter } from "../csv/application-csv-exporter"
 
+enum OrderByParamEnum {
```

Comment:
So far we have not been adding Enum suffix to the enums

Reply:
Fixed